marlowe-dashboard-client/tailwind.config.js

```diff
@@ -22,29 +17,13 @@ module.exports = {
       red: "#de4c51",
       transgray: "rgba(10,10,10,0.4)",
     },
-    spacing: {
-      0: "0px",
-      0.25: "0.25rem",
-      0.5: "0.5rem",
-      0.75: "0.75rem",
-      1: "1rem",
-      1.5: "1.5rem",
-      2: "2rem",
-      3: "3rem",
-      12: "12rem",
-      22: "22rem",
-    },
-    fontSize: {
-      xs: ["0.75rem", { lineHeight: "1rem" }],
-      sm: ["0.875rem", { lineHeight: "1.25rem" }],
-      base: ["1rem", { lineHeight: "1.5rem" }],
-      lg: ["1.125rem", { lineHeight: "1.75rem" }],
-      xl: ["1.5rem", { lineHeight: "1.75rem" }],
-      "2xl": ["2rem", { lineHeight: "1.75rem" }],
+    boxShadow: {
+      DEFAULT: "0 3px 6px 0 rgba(0, 0, 0, 0.21)"
```

Comment:
why did you removed the rest of the shadows? If we want only to have 2 options, DEFAULT and deep, then we should move `deep` here. If you didn't wanted to remove, then maybe moving DEFAULT to extend